tmuxp/cli.py

```diff
@@ -924,3 +925,11 @@ def command_convert(config):
                 buf.write(newconfig)
                 buf.close()
                 print('New config saved to <%s>.' % newfile)
+
+
+@cli.command(name='ls', short_help='List configured sessions in $HOME/.tmuxp dir.')
+def command_ls():
+    tmuxp_dir = Path.home() / '.tmuxp'
```

Comment:
We have `get_config_dir` to get this path.
```suggestion
    tmuxp_dir = get_config_dir()
```

It respects the user's [`XDG_CONFIG_HOME`](https://wiki.archlinux.org/index.php/XDG_Base_Directory#User_directories) and/or `TMUXP_CONFIGDIR` if its set